deployment/k8s/helm/victoria-metrics/values.yaml

```diff
@@ -215,9 +252,7 @@ vmstorage:
     subPath: ""
 
 
-  podAnnotations: {
-    prometheus.io/scrape: "true"
```

Comment:
Why this podAnnotation is removed?

Reply:
this is a redundant annotation. 
There are two types how k8s can inform prometheus about the service
1.  Three annotations of pod level 
```
 prometheus.io/path: /metrics
 prometheus.io/port: "80"
 prometheus.io/scrape: "true"
```
As you see port and path were missing for all templates
2. Install Prometheus Operator and define ServiceMonitror